package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "docz": "^0.12.9",
     "eslint": "^5.9.0",
+    "null": "^1.0.1",
     "react": "^16.6.0",
     "react-bootstrap": "^0.32.0",
     "react-dom": "^16.6.0",
-    "react-scripts": "2.1.1"
+    "react-scripts": "2.1.1",
+    "typescript": "*"
```

Comment:
Not sure if this is needed, but seems to suppress some of the warnings we were getting.